wafer/markdown.py

```diff
@@ -0,0 +1,10 @@
+import bleach
+from bleach_allowlist import markdown_tags, markdown_attrs
+from markdown import markdown
+
+
+def bleached_markdown(text, **kwargs):
```

Comment:
Maybe this should live in django-markitup?

Reply:
I'm not sure we want to build this into django-markitup directly. It's aiming to be renderer agnostic and so this would add a number of dependencies that not all consumers will want.

It certainly worth adding as an example to the django-markitup README, though.